vscode-build/vscode.patch

```diff
@@ -186,13 +199,14 @@ index 9cddae242a..35400d5d59 100644
 +    "applicationName": "code-web",
 +    "dataFolderName": ".vscode-web",
 +    "version": "1.53.0"
-+  }
++  },
++  "webviewEndpoint": "http://localhost:8000/vscode/vscode/vs/workbench/contrib/webview/browser/pre"
```

Comment:
Oops this line needs to go because it was moved to the `workbench.ts` file just below